README.md

```diff
@@ -252,11 +252,28 @@ To do this, just follow these simple steps:
 
 To destroy the provisioned infrastructure, just run `rake terraform:destroy`
 
+
 ## Installer tool
 
 ```
 ruby -e "$(curl -fsSL https://raw.githubusercontent.com/openware/opendax/master/bin/install)"
 ```
 
+
+## Using opendax deployment for local development
```

Comment:
Please update the docs regarding the `remoteAccess` field(describe how it affects the deployment and usage of the platform)